### ISSUE TITLE
"Add import" and autocomplete don't discover newly added packages

### ISSUE THREAD

**Issue by ahmetalpbalkan** [Microsoft/vscode-go#691]
If I add a new dependency to my vendor directory (say, using [govendor](https://github.com/kardianos/govendor)) or $GOPATH (using `go get`) the `> Go: Add Import` command does not list these packages.

I have to restart vscode to get those packages working in "Add Import" and autocomplete.

**Comment by ramya-rao-a** [comment]
@ahmetalpbalkan This works for me. Here is what I tried:

- Delete one of the packages I had from $GOPATH/bin, $GOPATH/src and $GOPATH/pkg
- Open VS Code and open a Go project, ensure `Add Import` does not show the deleted package
- Run `go get` to get the package
- In the VS Code window that was open, run `Add Import`
   - I was able to see the package in the list and add it to the Go file
- Autocomplete works for the newly added package in the Go file

The `Add import` command runs `gopkgs` tool to get list of installed packages from $GOPATH/pkg every single time. As in, there is no caching done. So, I don't see why it wouldnt show newly added packages
Autocomplete uses the `gocode` tool also makes use of installed packages from $GOPATH/pkg. 

The only way the package would be missing in `Add Import` list and from autocomplete is if $GOPATH/pkg didnt have the package. Restarting VS Code doesnt do any package installation in $GOPATH/pkg, so I am stumped how that fixes the issue for you.
